app/post.go

```diff
@@ -470,6 +554,13 @@ func (a *App) sendUpdatedPostEvent(post *model.Post) {
 	a.Publish(message)
 }
 
+func (a *App) sendRethreadedPostEvent(post *model.Post, originalRootId string) {
```

Comment:
Strictly speaking, do we need a new websocket event for this? Presumably we could just `sendUpdatedPostEvent`, and if the client has seen the post in question, it will already have the `original_root_id`, and otherwise doesn't care?

Reply:
Actually, I realize that we won't have the old post id in this context, so my comment as written doesn't quite apply. That being said, we already handle the equivalent of new posts and deleting old posts, but perhaps this new event is necessary to avoid the "deleted post" message. 1/5 on the idea of merging the websocket event, but I'm not sure what's needed.

Reply:
The websocket event used to update replycount in threads